pyanaconda/modules/users/users.py

```diff
@@ -215,10 +218,17 @@ def _ksdata_to_user_data(self, user_ksdata):
         # -> as user data is -1 by default ve only set it if kickstart has something,
         #    that is not None
         # We need to make sure we correctly convert between these two.
+        # We also cannot use values larger then the accepted range.
         if user_ksdata.uid is not None:
-            user_data.uid = user_ksdata.uid
+            user_data.uid = min(user_ksdata.uid, MAX_ID_VALUE)
+            user_data.uid_status = ""
+        else:
+            user_data.uid_status = ID_NOT_SET
         if user_ksdata.gid is not None:
-            user_data.gid = user_ksdata.gid
+            user_data.gid = min(user_ksdata.gid, MAX_ID_VALUE)
+            user_data.gid_status = ""
+        else:
+            user_data.gid_status = ID_NOT_SET
```

Comment:
Please, use the fact that `user_data` is a new instance with default values. You don't have to rewrite defaults with defaults.